### ISSUE TITLE
tentaclerape.net being weird (yeah I see the irony lol)

### ISSUE THREAD

**Issue by wtevrs** [Bionus/imgbrd-grabber#1581]
**Bug description**

When searching and/or downloading anything, tentaclerape.net says "No valid source of the site returned result". Then, using the "Get all" button, tentaclerape.net shows up and says it has millions of images to download.

**Steps to reproduce**

1. Search any tag. (Note: Only happens when you search with at least one other site, otherwise you can't click on "Get all" for obvious reasons.)
2. Click on 'Get all'
3. tentaclerape.net should be there.

**Expected behavior**

If the app's somehow being blocked by the site, at least don't try to download non existing images.

**Context**

* Provide your `main.log` file to see if any obvious error occured.
[main.log](https://github.com/Bionus/imgbrd-grabber/files/2844427/main.log)
* Providing your `settings.ini` file will greatly help maintainers reproduce your problem if it may be caused by your configuration.
"We don't support that file type" says GitHub...so here's a txt version:
[settings.txt](https://github.com/Bionus/imgbrd-grabber/files/2844507/settings.txt)

**Screenshots**

![1](https://user-images.githubusercontent.com/47420034/52470011-af23ef00-2b95-11e9-845c-0dd6d0aa6808.png)
![2](https://user-images.githubusercontent.com/47420034/52470015-b21edf80-2b95-11e9-891f-fee73609d2da.png)

**System information**

- OS: Windows 10
- Grabber version: 6.0.6

**Additional context**

If it's broken and I just can't use it, that's fine. Just wondering if there's a way to fix it.

**Comment by wtevrs** [comment]
After going through the site, it seems they just copy/paste the same set of tags for all their images...which makes them useless. I did however have the same problem with rule34.paheal.net, but I was able to fix it by changing the "Referer (default)" setting to "Image".

I didn't see a way to close or delete this post myself, so it can just be closed by who ever can.

Thanks for still working on this app btw...though I would love it if Hentai Foundry was a valid source lol.

**Comment by Bionus** [comment]
> After going through the site, it seems they just copy/paste the same set of tags for all their images...which makes them useless.

Yep, they have very few tags, so you can't really expect to use the tags from other sources there. 🤷‍♂️ 

> Thanks for still working on this app btw...though I would love it if Hentai Foundry was a valid source lol.

Feel free to open another ticket to suggest it, with information about use cases!

> I didn't see a way to close or delete this post myself, so it can just be closed by who ever can.

:+1:
